axelor-human-resource/src/main/java/com/axelor/apps/hr/service/allocation/AllocationLineComputeServiceImpl.java

```diff
@@ -211,4 +213,60 @@ protected BigDecimal computeProrata(
 
     return prorata;
   }
+
+  @Override
+  public BigDecimal getAllocatedTime(Project project, Sprint sprint) {
+    List<AllocationLine> allocationLineList =
+        allocationLineRepo.all().fetch().stream()
+            .filter(allocationLine -> allocationLine.getProject().equals(project))
+            .collect(Collectors.toList());
+    BigDecimal allocatedTime = BigDecimal.ZERO;
+    for (AllocationLine allocationLine : allocationLineList) {
```

Comment:
Check if allocationLine.getPeriod(), sprint.getFromDate() and sprint.getToDate() are not null to avoid npe in the loop